src/main/requests/page/linked_instagram_account/GetLinkedInstagramAccountRequest.ts

```diff
@@ -0,0 +1,42 @@
+import { AxiosResponse } from 'axios';
+import { AbstractRequest } from '../../AbstractRequest';
+import { GetLinkedInstagramAccountResponse } from './GetLinkedInstagramAccountResponse';
+
+/**
+ * A Get request to obtain instagram_business_account information related with
+ * a Facebock page
+ *
+ * @author Andres Gutierrez <andres99@gmail.com>
+ * @since `next.release`
+ */
+export class GetLinkedInstagramAccountRequest extends AbstractRequest<GetLinkedInstagramAccountResponse> {
+    // A business Instagram account has to be linked to a Facebook page. To get the ID
+    // of an Instagram account you need the ID of their linked Facebook page.
+    facebookPageId: string;
+
+    /**
+     * The constructor.
+     *
+     * @param accessToken the access token.
+     * @param facebookPageId Facebook page where Instagram Account is linked.
```

Comment:
Let's make it simple. From the name of the class people already get that this is all related to a linked Instagram page

```suggestion
     * @param facebookPageId the Facebook page id.
```